### ISSUE TITLE
sfdx force:mdapi:convert does not convert CustomLabels to source format.  No error message.

### ISSUE THREAD

**Issue by camerongoodman** [forcedotcom/cli#1540]
### Summary

Running **sfdx force:mdapi:convert** does not convert any CustomLabels to source format.  It skips over them.  This is causing a serious issue in some of our build pipelines that depend on this feature.

Using either
sfdx-cli/7.151.1 win32-x64 node-v14.19.1
sfdx-cli/7.151.1 linux-x64 node-v14.19.2

### Steps To Reproduce:

**Repository to reproduce:** [sfdx-cli-mdapi-convert-bug](https://github.com/camerongoodman/sfdx-cli-mdapi-convert-bug)

Steps:

1. convert original source to mdapi format `sfdx force:source:convert -r force-app -d mdapi`
1. observe CustomLabels in mdapi folder and package.xml
1. convert MDAPI back to SOURCE format `sfdx force:mdapi:convert -r mdapi -d src`
1. observe NOTHING was converted, `\src` folder is empty
1. Use legacy mdapi:convert `sfdx force:mdapi:legacy:convert -r mdapi -d src2`
1. Observe legacy convert works as expected in `\src2` folder

### Expected result

CustomLabels metadata should be converted into source format in the expected folder

### Actual result

CustomLabels do not get converted from MDAPI to SOURCE format.

### System Information
Recreated in Windows and Ubuntu
 CLI Version :
        sfdx-cli/7.151.1

 Architecture:
        win32-x64

 Node Version :
        node-v14.19.1

 Plugin Version:
        @oclif/plugin-autocomplete 0.3.0 (core)
        @oclif/plugin-commands 1.3.0 (core)
        @oclif/plugin-help 3.3.1 (core)
        @oclif/plugin-not-found 1.2.6 (core)
        @oclif/plugin-plugins 1.10.11 (core)
        @oclif/plugin-update 1.5.0 (core)
        @oclif/plugin-warn-if-update-available 1.7.3 (core)
        @oclif/plugin-which 1.0.4 (core)
        @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
        alias 2.0.0 (core)
        apex 0.11.0 (core)
        auth 2.0.2 (core)
        community 1.1.4 (core)
        config 1.4.6 (core)
        custom-metadata 1.1.0 (core)
        data 2.0.0 (core)
        generator 2.0.1 (core)
        info 2.0.0 (core)
        limits 2.0.0 (core)
        org 1.13.1 (core)
        salesforce-alm 54.4.0 (core)
        schema 2.1.0 (core)
        sfdx-cli 7.151.1 (core)
        sfpowerkit 4.2.8
        signups 1.1.0 (core)
        source 1.9.8 (core)
        telemetry 2.0.0 (core)
        templates 54.7.0 (core)
        trust 1.1.0 (core)
        user 1.7.1 (core)

 OS and Version:
        Windows_NT 10.0.22621

CLI Version : 
	sfdx-cli/7.151.1

 Architecture: 
	linux-x64

 Node Version : 
	node-v14.19.2

 Plugin Version: 
	@oclif/plugin-autocomplete 0.3.0 (core)
	@oclif/plugin-commands 1.3.0 (core)
	@oclif/plugin-help 3.3.1 (core)
	@oclif/plugin-not-found 1.2.6 (core)
	@oclif/plugin-plugins 1.10.11 (core)
	@oclif/plugin-update 1.5.0 (core)
	@oclif/plugin-warn-if-update-available 1.7.3 (core)
	@oclif/plugin-which 1.0.4 (core)
	@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
	alias 2.0.0 (core)
	apex 0.11.0 (core)
	auth 2.0.2 (core)
	community 1.1.4 (core)
	config 1.4.6 (core)
	custom-metadata 1.1.0 (core)
	data 2.0.0 (core)
	generator 2.0.1 (core)
	info 2.0.0 (core)
	limits 2.0.0 (core)
	org 1.13.1 (core)
	salesforce-alm 54.4.0 (core)
	schema 2.1.0 (core)
	sfdx-cli 7.151.1 (core)
	signups 1.1.0 (core)
	source 1.9.8 (core)
	telemetry 2.0.0 (core)
	templates 54.7.0 (core)
	trust 1.1.0 (core)
	user 1.7.1 (core)

 OS and Version: 
	Linux 5.15.0-30-generic

### Additional information

This is a high priority issue affecting our build pipeline.

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-11186731](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000xcDrOYAU/view)

**Comment by cristiand391** [comment]
Hey @camerongoodman, I was able to reproduce using latest-rc `sfdx-cli/7.152.0 darwin-x64 node-v16.13.2` too, thanks for the reproduction repo!

Unfortunately the command returns exit code 0 after failing to convert to source format, in the meantime please switch to the `legacy` command.

**Comment by sumit-hi** [comment]
Hey @camerongoodman any update on this, is this fixed yet?

**Comment by cristiand391** [comment]
Hi @sumit-hi we have a fix open (link above), once it passes QA it should be available in the the release candidate.

**Comment by camerongoodman** [comment]
Tested in 7.160 and seems to be resolved now

**Comment by thomasminney** [comment]
Tested in 7.161 and this is not resolved if the existing CustomLabels.meta-xml does not exactly match the file that would be created by force:mdapi:convert on the mdapi/CustomLabels.labels file:

`sfdx force:mdapi:convert -r md --json`
```
{
  "status": 0,
  "result": [
    {
      "fullName": "CustomLabels",
      "type": "CustomLabels",
      "filePath": "md\\labels\\CustomLabels.labels",
      "state": "Add"
    }
  ]
}
```

Whereas the legacy command creates a usable .dup file

`sfdx force:mdapi:legacy:convert -r md --json`
```
{
  "status": 0,
  "result": [
    {
      "fullName": "CustomLabels",
      "type": "CustomLabels",
      "filePath": "src\\main\\default\\labels\\CustomLabels.labels-meta.xml.dup",
      "state": "Duplicate"
    }
  ],
  "warnings": [
    "The command \"MdapiConvertCommand\" has been deprecated and will be removed in v56.0 or later. Use \"force:mdapi:convert\" instead."
  ]
}
```

**Comment by peternhale** [comment]
@thomasminney would you please provide the md source that you are using.

**Comment by peternhale** [comment]
fix in version 7.162.0 of sfdx-cli

**Comment by thomasminney** [comment]
@peternhale tested in 7.162.0 and `force:mdapi:convert` does now convert custom labels, thank you.
